addons/web/static/src/views/view_components/multi_currency_popover.js

```diff
@@ -33,8 +33,9 @@ export class MultiCurrencyPopover extends Component {
                 currencies.push({
                     ...getCurrency(currencyId),
                     id: currencyId,
-                    rate: this.state.rates[currencyId],
-                    value: this.props.value / this.state.rates[currencyId],
+                    rate: this.state.rates[currencyId].rate,
+                    date: this.state.rates[currencyId].date,
```

Comment:
regarding my comment above:
```suggestion
                    date: toLocaleDateString(this.state.rates[currencyId].date),
```